cmake/configs/default.cmake

```diff
@@ -92,6 +92,7 @@ hunter_config(ZeroMQ VERSION 4.1.4-p2)
 hunter_config(caffe VERSION rc3-p2)
 hunter_config(ccv VERSION 0.7-p6)
 hunter_config(cereal VERSION 1.1.2-p5)
+hunter_config(ceres-solver VERSION 1.12.0)
```

Comment:
Use [jenkins.py](https://github.com/ruslo/hunter/wiki/dev.hunter.root#jenkinspy) testing to avoid such issues.